### PR TITLE
Add cstorpool failure and add the application

### PR DESCRIPTION
**Pull request by inyee786**
**What this PR does / why we need it**:

- Add cstor pool failure

- remove cockraochdb percona redis and postresql under jiva namespaces

Signed-off-by: inyee786 <intakhab.cusat@gmail.com>

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
screenshot from 2018-11-20 18-24-05
screenshot from 2018-11-20 18-24-19
screenshot from 2018-11-21 15-43-33

### CODE REVIEW

File: src/app/workload-dashboard/workload-dashboard.component.ts [hunk 1]
```diff
@@ -32,39 +36,53 @@ export class WorkloadDashboardComponent implements OnInit {
...
       this.kubernetsServices.getAllstatus("redis-jiva").subscribe(res => {
         this.redisjivaStatus = res.status;
       });
+      this.kubernetsServices.getAllstatus("redis-cstor").subscribe(res => {
+        this.rediscstorStatus = res.status;
+      });
       this.kubernetsServices.getAllstatus("postgresql-jiva").subscribe(res => {
         this.postgresqljivaStatus = res.status;
       });
+      this.kubernetsServices.getAllstatus("postgresql-jiva").subscribe(res => {
```
Comment (bhaskarhc) [inline src/app/workload-dashboard/workload-dashboard.component.ts]: postgresql-cstor

File: src/app/workload-dashboard/workload-dashboard.component.html [hunk 2]
```diff
@@ -320,6 +384,69 @@ <h5 style="font-size: 0.8em;text-align: left;padding-left: 14px;">Applications
...
+                        <div class="d-flex bg-light p-3">
+                          <div class="d-flex justify-content-between w-100">
+                            <div>
+                              <img src="/assets/images/workloads-logos/cockroachdb.svg" alt="" style="height:50px;width:50px">
+                              <p class="mb-2">CockroachDB</p>
+                              <p>2 Replicas </p>
+                            </div>
+                            <div>
+                              <img src="/assets/images/company-logos/mule-without-tagline.png" alt="openebs" style="height:50px;width:50px">
+                              <p class="mb-2">Jiva 0.7 </p>
```
Comment (bhaskarhc) [inline src/app/workload-dashboard/workload-dashboard.component.html]: cstor 0.7

File: src/app/workload-dashboard/workload-dashboard.component.html [hunk 3]
```diff
@@ -320,6 +384,69 @@ <h5 style="font-size: 0.8em;text-align: left;padding-left: 14px;">Applications
...
+                        <div class="d-flex bg-light p-3">
+                          <div class="d-flex justify-content-between w-100">
+                            <div>
+                              <img src="/assets/images/workloads-logos/redis.svg" alt="" style="height:50px;width:50px">
+                              <p class="mb-2">Redis</p>
+                              <p>2 Replicas </p>
+                            </div>
+                            <div>
+                              <img src="/assets/images/company-logos/mule-without-tagline.png" alt="openebs" style="height:50px;width:50px">
+                              <p class="mb-2">Jiva 0.7 </p>
```
Comment (bhaskarhc) [inline src/app/workload-dashboard/workload-dashboard.component.html]: cstor 0.7

File: src/app/workload-dashboard/workload-dashboard.component.ts (hunk 1 above)
Comment (inyee786) [reply src/app/workload-dashboard/workload-dashboard.component.ts]: done

File: src/app/workload-dashboard/workload-dashboard.component.html (hunk 2 above)
Comment (inyee786) [reply src/app/workload-dashboard/workload-dashboard.component.html]: done

File: src/app/workload-dashboard/workload-dashboard.component.html (hunk 3 above)
Comment (inyee786) [reply src/app/workload-dashboard/workload-dashboard.component.html]: done

**Pull request merged**